.changeset/rare-panthers-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+"@chakra-ui/styled-system": major
+---
+
+Union types changed to a more understandable
```

Comment:
I think you wanted to write `Union types changed to be more understandable` instead.